source/docs/api-reference/api-usage/faults.rst

```diff
@@ -5,8 +5,40 @@ Device Faults
 
 Faults are stored in two fashions. There are "live" faults, which are reported in real-time, and "sticky" faults, which assert persistently and stay asserted until they are manually cleared (like trouble codes in a vehicle).
 
-Sticky Faults can be cleared in Phoenix Tuner X, while a regular fault can only be cleared when the offending problem has been resolved. Faults can be cleared via the :guilabel:`Blink/Clear Faults` button.
+Sticky Faults can be cleared in Phoenix Tuner X or via the ``clearStickyFaults()`` function on the offending device, while a regular fault can only be cleared when the offending problem has been resolved. Faults can be cleared via the :guilabel:`Blink/Clear Faults` button.
 
 .. image:: images/self-test-clear-faults.png
    :width: 70%
    :alt: Blink/Clear faults button in tuner located in the bottom right.
+
+Retrieving Faults via API
+-------------------------
+
+Faults can also be retrieved via the API using the ``getFault_XXX()`` (regular) or ``getStickyFault_XXX()`` (sticky) functions on the offending device. ``XXX`` can be replaced with the name of the fault. This can be useful for diagnostics or error handling.
```

Comment:
"Faults can also be retrieved in API using the \`\`get(Sticky)Fault_*()\`\` methods on the device object. This can be useful for diagnostics or error handling."

Reply:
I dislike the ``get(Sticky)Fault_*()`` as it seems jarring, but I can modify the latter half of the first sentence and add the suggested sentence.

Reply:
If we don't like `get(Sticky)Fault_*()`, then I guess you can instead write: "using the \`\`getFault_*()\`\` (regular) and \`\`getStickyFault_*()\`\` (sticky) methods on the device object."